protocol/communities/manager.go

```diff
@@ -88,6 +97,11 @@ func NewManager(identity *ecdsa.PublicKey, db *sql.DB, logger *zap.Logger, verif
 	return manager, nil
 }
 
+func (m *Manager) LogStdout(msg string, fields ...zap.Field) {
```

Comment:
I think that the name of the func is misleading because it logs to both std and logger.
Maybe LogAndStdout will be slightly better ?

Reply:
Actually, that commit has landed already last week. Sorry. Need to rebase this one